spec/features/friend_request_spec.rb

```diff
@@ -0,0 +1,56 @@
+require 'rails_helper'
+
+RSpec.describe 'Friend Request', type: :feature do
+  user1 = ['user1', 'user1@test.com', 'password1']
```

Comment:
- **[OPTIONAL]** We can clean this setup using `factory_bot` gem, which gives you ability to generate fast test objects.